ale/base/data_naif.py

```diff
@@ -426,6 +439,39 @@ def detector_center_line(self):
         """
         return float(spice.gdpool('INS{}_BORESIGHT_LINE'.format(self.ikid), 0, 1)[0])
 
+    @property
+    def swap_observer_target(self):
+        """
+        Returns if the observer and target should be swapped when determining the
+        sensor state relative to the target. This is defined by a keyword in
+        ISIS IAKs. If the keyword is not defined in any loaded kernels then False
+        is returned.
+
+        Expects ikid to be defined. This should be an integer containing the
+        Naif Id code of the instrument.
+        """
+        try:
+            swap = spice.gcpool('INS{}_SWAP_OBSERVER_TARGET'.format(self.ikid), 0, 1)[0]
+            return swap.upper() == "TRUE"
+        except:
+            return False
```

Comment:
The only problem I have with theses gcpool calls is that they are dependent on having the ISIS addendum kernel loaded into memory which isn't the case if you're using a Pds3 image. So for any pds3 image you'll get a position or velocity in one direction and in ISIS you'll get the swapped result.

Reply:
This is a good point. A few followup questions: for input pds3 images, are we expecting ale's output to be supported in ISIS, or just csm? I know the option of `to_isis` exists, but what is the current goal?  If it's just csm, then doesn't the ISIS camera model take the swapping into account? So you'd get positions and velocities swapped in the isds between the output of ale's `to_isis` and `to_usgscsm` formatters, but would ultimately end up with matching (or at least close) positions out of campt and csm's image-to-ground call?

Reply:
This is potentially a side-note: Would we ever want the ISDs generated by `to_isis` vs `to_usgscsm to output different exterior orientation values due to different swap_observer_target settings between ISIS and CSM? i.e. Since the ISIS camera model incorporates the swap, but presumably the CSM camera model doesn't(?) would it make sense for these numbers to be different, but then ultimately end up with matching (or at least close) positions out of campt and csm's image-to-ground call?

Reply:
We could change this to default to returning True because it's the correct answer in 99% of cases (Active sensors like Radar being the exception) and then update all of the ISIS IAKs to have FALSE in them.

Reply:
We can actually start updating IAKs right now with the currently hard-coded defaults in ISIS. It shouldn't make any differences in current ISIS.